reporters_db/data/reporters.json

```diff
@@ -1585,18 +2509,79 @@
             }
         }
     ],
-    "CIT": [
+    "Chi. Leg. News": [
         {
-            "cite_type": "neutral",
+            "cite_type": "federal",
             "editions": {
-                "CIT": {
+                "Chi. Leg. News": {
+                    "end": "1925-12-31T00:00:00",
+                    "start": "1868-01-01T00:00:00"
+                }
+            },
+            "mlz_jurisdiction": [
+                "us:c7:il.nd;district.court"
+            ],
+            "name": "Chicago Legal News (1868-1925) (Illinois)",
+            "variations": {
+                "placeholder_0": "Chi. Leg. News"
```

Comment:
This is...odd?

Reply:
yeah - removed-  this is an artifact from my adding script - no others existed.